src/Tools/dotnet-stack/Symbolicate.cs

```diff
@@ -285,7 +314,7 @@ private static string GetLineFromMetadata(MetadataReader reader, string line, St
                 Argument = new Argument<FileInfo>(name: "output-path")
                 {
                     Arity = ArgumentArity.ZeroOrOne
-                }
+                }.ExistingOnly()
```

Comment:
I'm not familiar with this method. Is this saying that the output file needs to exist in order for the option to be valid? That seems opposite of the intention of the option, i.e., create the file and fill it with the contents we generate.

Reply:
Yes, this will generate an error if the file doesn't exist.

Reply:
Oh. It's my mistake. I'll delete it.